components/theme/colors.js

```diff
@@ -15,153 +15,153 @@ let colors = {
 // (https://yeun.github.io/open-color/), with the exception of blue[0-9] which
 // are hand picked
 colors.palette = {
-  "base": "#0588cb",
-  "black": "#212529",
-  "gray0": "#f8f9fa",
-  "gray1": "#f1f3f5",
-  "gray2": "#e9ecef",
-  "gray3": "#dee2e6",
-  "gray4": "#ced4da",
-  "gray5": "#adb5bd",
-  "gray6": "#868e96",
-  "gray7": "#495057",
-  "gray8": "#343a40",
-  "gray9": "#212529",
-  "blue0": "#e7f5ff",
-  "blue1": "#c9e8ff",
-  "blue2": "#94d1ff",
-  "blue3": "#5db8fe",
-  "blue4": "#37a6ed",
-  "blue5": "#0588cb",
-  "blue6": "#0f77b8",
-  "blue7": "#056aa6",
-  "blue8": "#005f9c",
-  "blue9": "#005a99",
-  "indigo0": "#edf2ff",
-  "indigo1": "#dbe4ff",
-  "indigo2": "#bac8ff",
-  "indigo3": "#91a7ff",
-  "indigo4": "#748ffc",
-  "indigo5": "#5c7cfa",
-  "indigo6": "#4c6ef5",
-  "indigo7": "#4263eb",
-  "indigo8": "#3b5bdb",
-  "indigo9": "#364fc7",
-  "violet0": "#f3f0ff",
-  "violet1": "#e5dbff",
-  "violet2": "#d0bfff",
-  "violet3": "#b197fc",
-  "violet4": "#9775fa",
-  "violet5": "#845ef7",
-  "violet6": "#7950f2",
-  "violet7": "#7048e8",
-  "violet8": "#6741d9",
-  "violet9": "#5f3dc4",
+  'base': '#0588cb',
+  'black': '#212529',
+  'gray0': '#f8f9fa',
+  'gray1': '#f1f3f5',
+  'gray2': '#e9ecef',
+  'gray3': '#dee2e6',
+  'gray4': '#ced4da',
+  'gray5': '#adb5bd',
+  'gray6': '#868e96',
+  'gray7': '#495057',
+  'gray8': '#343a40',
+  'gray9': '#212529',
+  'blue0': '#e7f5ff',
+  'blue1': '#c9e8ff',
+  'blue2': '#94d1ff',
+  'blue3': '#5db8fe',
+  'blue4': '#37a6ed',
+  'blue5': '#0588cb',
+  'blue6': '#0f77b8',
+  'blue7': '#056aa6',
+  'blue8': '#005f9c',
+  'blue9': '#005a99',
+  'indigo0': '#edf2ff',
+  'indigo1': '#dbe4ff',
+  'indigo2': '#bac8ff',
+  'indigo3': '#91a7ff',
+  'indigo4': '#748ffc',
+  'indigo5': '#5c7cfa',
+  'indigo6': '#4c6ef5',
+  'indigo7': '#4263eb',
+  'indigo8': '#3b5bdb',
+  'indigo9': '#364fc7',
+  'violet0': '#f3f0ff',
+  'violet1': '#e5dbff',
+  'violet2': '#d0bfff',
+  'violet3': '#b197fc',
+  'violet4': '#9775fa',
+  'violet5': '#845ef7',
+  'violet6': '#7950f2',
+  'violet7': '#7048e8',
+  'violet8': '#6741d9',
+  'violet9': '#5f3dc4',
 
   // XXX drop these once we check that nothing breaks
-  "fuschia0": "#f8f0fc",
-  "fuschia1": "#f3d9fa",
-  "fuschia2": "#eebefa",
-  "fuschia3": "#e599f7",
-  "fuschia4": "#da77f2",
-  "fuschia5": "#cc5de8",
-  "fuschia6": "#be4bdb",
-  "fuschia7": "#ae3ec9",
-  "fuschia8": "#9c36b5",
-  "fuschia9": "#862e9c",
+  'fuschia0': '#f8f0fc',
+  'fuschia1': '#f3d9fa',
+  'fuschia2': '#eebefa',
+  'fuschia3': '#e599f7',
+  'fuschia4': '#da77f2',
+  'fuschia5': '#cc5de8',
+  'fuschia6': '#be4bdb',
+  'fuschia7': '#ae3ec9',
+  'fuschia8': '#9c36b5',
+  'fuschia9': '#862e9c',
```

Comment:
We should probably eventually just drop, but only after have done a check to ensure it's not referenced anywhere (this is an artifact from a typo in spelling of the color [fuchsia](https://en.wikipedia.org/wiki/Fuchsia_(color)))

Reply:
We are not the only ones who made this mistake. There are 35K results for `fuschia` on Github. And here's [the reason](https://en.wikipedia.org/wiki/Fuchsia#Pronunciation_and_spelling) why:

> the word's German origin is "fook-sya" /ˈfʊksija/, the standard pronunciation for the common name in English is "fyu-sha" /ˈfjuːʃə/. As a consequence, fuchsia is often misspelled as fuschia in English.

Reply:
`fuschia` is only referenced in two repos, once each. In [explorer](https://github.com/ooni/explorer/blob/master/components/test-info.js#L26) and [probe-desktop](https://github.com/ooni/probe-desktop/blob/master/renderer/components/nettests/performance/index.js#L14).

If we drop these in this version, it breaks compatibility and hence major version upgrade?

Reply:
Never mind. We will do this separately. Ticket opened #44